CONTRIBUTING.md

```diff
@@ -86,15 +86,35 @@ the VCS/git logs more valuable. The general structure of a commit message is as
 [optional footer(s)]
 ```
 
-- **Prefix the commit subject with a _type_:** `doc:`, `test:`
-  `runtime:`, ...
-    - Subject line for commits with only style/lint changes can be a single
-      word: `style` or `lint`.
-- **Add the optional scope following <type> if possible:** `(lsp)`, `(treesitter)`, `(multigrid)`, ...
+- Prefix the commit subject with one of the following _types_:
+    - `build`: changes related to the build system (involving scripts, configurations or tools) and package dependencies.
+    - `ci`: changes related to the continuous integration and deployment system - involving scripts, configurations or tools.
+    - `docs`: all documentation changes. This includes both external documentation intended for end users as well as internal documentation intended for developers.
+    - `feat`: new abilities or functionality in the product end users can utilize.
+    - `fix`: a bug fix in the product that affects end users.
```

Comment:
same here.

In fact, maybe just remove the "end users" altogether: 
* "new abilities and functionality"
* "performance improvements"

Reply:
Thanks for the feedback.

I've applied the changes. I wasn't satisfied with the "end users" expression either but couldn't come up with a better way to phrase it.

I still wonder if we should somehow specify that `feat`, `fix` and `perf` types is only for changes that affect users to eliminate misunderstandings. A bug fix in the CI-chain or a PVS warning fix should not use the `fix` type.

Reply:
You could make clear for `ci` that "all changes" are meant? Otherwise I wouldn't sweat it too much. Commit messages can always be amended.